tensorrec/tensorrec.py

```diff
@@ -635,6 +637,18 @@ def predict_item_representation(self, item_features):
         item_repr = self.tf_item_representation.eval(session=get_session(), feed_dict=feed_dict)
         return item_repr
 
+    def predict_user_bias(self, user_features):
+        # TODO: docstring
+        feed_dict = self._create_user_feed_dict(user_features_matrix=user_features)
+        predictions = self.tf_projected_user_biases.eval(session=get_session(), feed_dict=feed_dict)
+        return predictions
+
+    def predict_item_bias(self, item_features):
+        # TODO: docstring
+        feed_dict = self._create_item_feed_dict(item_features_matrix=item_features)
+        predictions = self.tf_projected_item_biases.eval(session=get_session(), feed_dict=feed_dict)
+        return predictions
```

Comment:
This is all copypasta from similar methods, so I'm not sure that this is doing what it should.

Reply:
I think it would be good for these methods to raise a NotImplementedError if the model isn't biased. That would avoid something like "`TensorRec` has no attribute `tf_projected_item_biases`."